pkg/controller/master/upgrade/upgrade_repo.go

```diff
@@ -204,6 +204,7 @@ func (r *Repo) createVM(image *harvesterv1.VirtualMachineImage) (*kubevirtv1.Vir
 							Cores:   1,
 							Sockets: 1,
 							Threads: 1,
+							Model:   "host-passthrough",
```

Comment:
Refer the definition `CPUModeHostPassthrough` ?

```
vendor/kubevirt.io/api/core/v1/schema.go:	CPUModeHostPassthrough                 = "host-passthrough"
vendor/kubevirt.io/kubevirt/pkg/virt-controller/services/nodeselectorrenderer.go:	if nsr.cpuModelLabel != "" && nsr.cpuModelLabel != cpuModelLabel(v1.CPUModeHostModel) && nsr.cpuModelLabel != cpuModelLabel(v1.CPUModeHostPassthrough) {
vendor/kubevirt.io/kubevirt/pkg/defaults/arm64.go:	defaultCPUModelArm64 = v1.CPUModeHostPassthrough
```

Reply:
@starbops The latest code still seems to use a string.

Reply:
Sorry I made a mistake. I should be fine now.